astropy/time/tests/test_methods.py

```diff
@@ -620,6 +629,37 @@ def test_sort(self, use_mask):
             assert np.all(self.t0.sort(-1)[:, :, 0] == self.t0.min(-1))
             assert np.all(self.t0.sort(-1)[:, :, -1] == self.t0.max(-1))
 
+    @pytest.mark.parametrize('axis', [None, 0, 1, 2, (0, 1)])
```

Comment:
Yeah no problem. I removed it because it was failing due to the TAI vs. UTC difficulty you pointed out. I'll just set the scale to TAI and hopefully it will be sufficient